src/test/scala/cognite/spark/v1/AssetsRelationTest.scala

```diff
@@ -367,13 +370,14 @@ class AssetsRelationTest extends FlatSpec with Matchers with ParallelTestExecuti
                 |lastUpdatedTime,
                 |0 as rootId,
                 |null as aggregates,
+                |null as labels,
```

Comment:
labels is a nullable field and I did not add this into schemas here but it is available in scala sdk with my merged PR. If I don't add "null as labels" in some test queries, it gives an error. As a user in transformations, I was skipping some nullable parameters in my queries. That was working there, do we handle this in jetfire when user skips some?

I am thinking if this will cause some compatibility issues. If so, I would prefer deleting "labels" from Assets in scala-sdk and take it as a separate PR along with spark part. I don't prefer to add it to spark-datasource in this PR since it would take time (because this was the hard part of all and I have several days to work in this sprint, I want to finish relationships primarily).

Reply:
@hakontro reminder

Reply:
I think it should be fine, can you try to add some labels to the tests and see if they still work?

Reply:
I added support for create and read operations. I saved upsert/update for later since it is not a setter. We can add/remove labels so I need to think how we want to do it and it can come later separately.  Added a couple of changes to the tests to address these changes. I left a question there in the tests.